src/actions/contractActions.ts

```diff
@@ -162,27 +166,16 @@ export function fetchContract(hash: string, populateStates = true) {
       dispatch(requestContract(hash))
 
       try {
-        const response = await fetch(`${GENERATE_BASE_URL()}/contract/${hash}`)
-
-        const json = await response.json()
-
-        if (populateStates) {
-          const invocationStatsResponse = await fetch(
-            `${GENERATE_BASE_URL()}/contract_stats/${hash}`,
-          )
-
-          const invocationStats = await invocationStatsResponse
-            .json()
-            .catch(error => {
-              console.error('An error occurred fetching invocation stats.', {
-                error,
-              })
-            })
-
-          json.invocationStats = invocationStats || null
-        }
+        const network = store.getState().network.network
+        const contract = await NeoRest.contract(hash, network)
+        const stats = await NeoRest.contractStats(hash, network)
 
-        dispatch(requestContractSuccess(hash, json))
+        dispatch(
+          requestContractSuccess(hash, {
+            contract,
+            stats: stats as InvocationStat,
```

Comment:
because the response of `NeoRest.contractStats()` is `Promise<Object>` but the format matches.